src/Stoa.ts

```diff
@@ -21,11 +22,64 @@ class Stoa {
             res.send(req.params.address + ':' + req.query.height);
         });
 
+        /**
+         * When a request is received through the `/push` handler
+         * JSON block data is parsed and stored on each storage.
+         */
         this.stoa.post("/push",
             (req: express.Request, res: express.Response, next: express.NextFunction) => {
-            console.log(req.body);
-            res.status(200).send();
-        });
+            var data: any = req.body;
+            var block: any;
+
+            if (data.block == undefined)
+            {
+                res.status(400).send("Block data is not defined");
+                return;
+            }
+
+            try {
+                block = JSON.parse(data.block);
+            } catch(e) {
+                res.status(400).send("Not a valid JSON format");
+                return;
+            }
+
+            console.log(block);
+            var blockStorage: BlockStorage = new BlockStorage((err: any) =>
```

Comment:
Why do you new the `BlockStorage` every time ? There should only be one needed.

Reply:
This issue has been resolved in #16.